e2e/packages/test-data/generateLogs.ts

```diff
@@ -55,12 +73,10 @@ export async function generateLogs(numRecords: number, rpc: string) {
     walletClient,
   });
 
-  console.log("calling setNumber");
-  for (let i = 0; i < numRecords - 1; i++) {
-    await worldContract.write.setNumber([i, i]);
-  }
-
-  const lastTx = await worldContract.write.setNumber([numRecords, numRecords]);
+  console.log("calling set");
+  await worldContract.write.set([[420]]);
+  console.log("calling push");
+  const lastTx = await transactionHook(worldContract);
```

Comment:
any reason we're still doing the `.set` above? instead of just letting the logic live within `transactionHook`?

Reply:
That was a mistake, ta